src/Twig/AppIntlExtension.php

```diff
@@ -12,41 +11,7 @@ class AppIntlExtension extends AbstractExtension
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('list_united_nations', [__CLASS__, 'getUnitedNationsList'], [
-                'needs_context' => true,
-            ]),
-            new TwigFunction('get_cities_names_from_insee_code', [__CLASS__, 'getCitiesNamesFromInseeCode']),
-            new TwigFunction('get_city_name_from_insee_code', [__CLASS__, 'getCityNameFromInseeCode']),
-            new TwigFunction('get_city_data_from_insee_code', [FranceCitiesBundle::class, 'getCityDataFromInseeCode']),
+            new TwigFunction('get_city_data_from_insee_code', [FranceCities::class, 'getCityByInseeCode']),
```

Comment:
we should check all usages of `get_city_data_from_insee_code` in templates and verify usage of `get_city_data_from_insee_code().postal_code` because it's an array now and not string